sdk/identity/identity/package.json

```diff
@@ -52,7 +52,8 @@
     "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "echo skipped",
     "integration-test:node": "dev-tool run test:node-js-input -- --timeout 180000 'dist-esm/test/public/node/*.spec.js' 'dist-esm/test/internal/node/*.spec.js'",
-    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "integration-test:playwright": "rimraf dist-playwright && npx playwright install && tsc -p playwright.tsconfig.json && rollup --config && playwright test -c dist-playwright",
```

Comment:
`npx playwright install` ensures that the web browser needed for testing is installed. Playwright doesn’t do that by default when you install it.